src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -94,6 +96,10 @@ def train(self, runner):
                 last_return = self.train_once(runner.step_itr,
                                               runner.step_path)
                 if cycle == 0 and self.evaluate:
```

Comment:
hmm is `self.evaluate` a constructor parameter? I think we should be evaluating all the time, not much reason to turn if off ever